content/docs/apis.md

```diff
@@ -2,57 +2,195 @@
 title: APIs
 description: Interact with TiKV using the raw key-value API or the transactional key-value API
 weight: 4
-draft: true
+draft: false
 ---
 
 TiKV offers two APIs that you can use to interact with it:
```

Comment:
```suggestion
TiKV offers two APIs that you can interact with:
```

Reply:
Thanks! You're right, this is better. :)